test/tools/cli/OutputFormat.java

```diff
@@ -42,6 +42,12 @@ public IonWriter createIonWriter(OutputStream outputStream) {
         public IonWriter createIonWriter(OutputStream outputStream) {
             return IonTextWriterBuilder.pretty().build(outputStream);
         }
+    },
+    NONE {
+        @Override
+        public IonWriter createIonWriter(OutputStream outputStream) {
+            return null;
```

Comment:
This should be an IonWriter that writes to nowhere. We still want to exercise all the logic of an IonWriter; we just don't want the data to be written anywhere. You can make a trivial `OutputStream` implementation to achieve this.

Reply:
Created a trivial OutputStream for none type. (IonProcess.java 245-247) 

I made a help function that returns an outputStream (IonProcess.java 241). Even though I used try-with-resource block to close the returned outputStream, do I need to close the trivial outputStream within the help function I made? (ionProcess.java 246)